coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "line": 100.0
+  }
+}
```

Comment:
@AkshayPashnathSuresh 
what is this file doing?

Reply:
@AkshayPashnathSuresh 
obviously SimpleCov configuration is not correct. Please fix it 🙏 
expected test coverage should look like this:

<img width="1272" alt="Screenshot 2021-05-25 at 8 01 55 PM" src="https://user-images.githubusercontent.com/25289449/119487315-3102ab00-bd94-11eb-9c54-85d2b031d069.png">